src/Fixer/FunctionNotation/PhpdocToReturnTypeFixer.php

```diff
@@ -42,7 +42,6 @@ final class PhpdocToReturnTypeFixer extends AbstractPhpdocToTypeDeclarationFixer
      * @var array<string, true>
      */
     private array $skippedTypes = [
-        'mixed' => true,
```

Comment:
At the beginning of `applyFix()`, there is a check to unset this value. The check is now obsolete and can be dropped.

Reply:
FYI: We can safely remove `'mixed' => true,` from skipped types, because it's handled in  [`AbstractPhpdocToTypeDeclarationFixer::getCommonTypeFromAnnotation()`](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/5f102b43616c746d1fca3f81b98873674d4f9708/src/AbstractPhpdocToTypeDeclarationFixer.php#L196-L198) anyway (for PHP <8 it will return `null` and fix won't be applied. That's why it just works in this PR even though there are no changes in the fixers, but only in test cases (which was confusing for me).

That condition mentioned by @julienfalque can be removed too.